src/projective.jl

```diff
@@ -339,8 +339,27 @@ function imagept2plane(C::Camera, xy::Array, planeP::Vector, planeN::Vector)
     # Subtract the tangential distortion components and divide by the radial
     # distortion factor to get an approximation of the undistorted normalised
     # image coordinates (with no skew)
-    x_n = (x_d .- dtx)./r_d
-    y_n = (y_d .- dty)./r_d
+
+    # inverse of the radial distortion is calculated by iteration 
+    # (see e.g., Drap&Lefèvre 2015, doi: 10.3390/s16060807)
+    # non-convergence of the mothod happens if an image point has no corresponding 
+    # point on the plane, due to distortion. This is not caught currently
+    
+    x_n=deepcopy(x_d)
+    y_n=deepcopy(y_d)
```

Comment:
`deepcopy` is used only when you request a recursive copy. For plain structure and plain numerical array, we can simply use `copy` here.

```suggestion
    x_n = copy(x_d)
    y_n = copy(y_d)
```